io.github.leonardschardijn.Chirurgien.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "io.github.leonardschardijn.Chirurgien",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "chirurgien",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home"
```

Comment:
Any chance it could be changed to use https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html? That will make it possible to drop this line.

Reply:
Oh, that's interesting. I completely missed that.
I've changed the commit that the v0.1 tag points to.